projects/storefrontapp-e2e-cypress/cypress/integration/regression/cart/cart.e2e-spec.ts

```diff
@@ -233,26 +236,31 @@ describe('Cart', () => {
       });
 
       it('should use existing cart when adding new entries', () => {
+        cart.registerCartRefreshRoute();
+
         cy.visit(`/product/${cart.products[0].code}`);
         cy.get('cx-breadcrumb h1').contains(cart.products[0].name);
         cart.clickAddToCart();
+        cy.wait('@refresh_cart').its('response.statusCode').should('eq', 200);
         cart.checkAddedToCartDialog();
         cy.visit(`/product/${cart.products[1].code}`);
         cy.get('cx-breadcrumb h1').contains(cart.products[1].name);
         cart.clickAddToCart();
+        cy.wait('@refresh_cart').its('response.statusCode').should('eq', 200);
         cart.checkAddedToCartDialog(2);
 
         cy.visit('/cart');
         cart.checkProductInCart(cart.products[0]);
         cart.checkProductInCart(cart.products[1]);
 
-        // cleanup
-        cart.registerCartRefreshRoute();
         cart.removeCartItem(cart.products[0]);
-        cy.wait('@refresh_cart');
+        cy.wait('@refresh_cart').its('response.statusCode').should('eq', 200);
+
+        // flakes due to button still being disabled.
+        cy.wait(3000);
```

Comment:
can we wait button enabled, instead of wait for 3s?

Reply:
it still failed even with that added 'guard'

Reply:
![Screen Shot 2022-10-26 at 1 45 29 PM](https://user-images.githubusercontent.com/29099466/198098914-fe789db0-829a-4cbf-9227-5215579ffa3b.png)

https://dashboard.cypress.io/projects/k3nmep/runs/34751/overview/3ab781db-9136-4b9e-8cf2-ce75c8acd91e

Reply:
reason I added a static wait was to let those other async actions to 'stabilize' before it gets to the button.